Dev-Corner/slides/tool_integration.html

```diff
@@ -45,65 +45,1274 @@
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            <img src="../images/conda_logo.png" class="no-border" />
+                            > Package, dependency and environment management
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            <img src="../images/conda_logo.png" class="no-border"  height=25 />
+                            #### Miniconda
+
+                            <img src="../images/miniconda_vs_anaconda.png" alt="Miniconda" class="no-border" height="500"/>
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            <img src="../images/planemo-logo.png" alt="Planemo" class="no-border" height=25/>
+                            #### Test Galaxy Tools using Conda packages
+
+                            <img src="../images/planemo_tool_building.png" alt="Testing tools" class="no-border"/>
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Hands-on
+
+                            <img src="../images/exercise.png" alt="exercise"/>
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Hands-on
+
+                            #### The Goal
+                            Use Conda package installation during the functional test of the wrapper for Seqtk
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Hands-on
+
+                            #### Before
+                            1. `conda uninstall seqtk`
+
+                        </script>
+                    </section>
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Hands-on
+
+                            #### Steps
+                            1. Fill a `<requirement>` tag to target a `seqtk` conda package
+                            2. Use `planemo conda_install` to install tools and set an environment
+                            3. Use `planemo test --conda_dependency_resolution ...`
+
+                        </script>
+                    </section>
+
+
+
+                    <section data-markdown>
+                        <script type="text/template">
+                            <img src="../images/bioconda.png" class="no-border"  height=25 />
+                            #### Bioconda
```

Comment:
It's just a couple of slides to introduce the channel Bioconda and the fact that we can easily contribute.